[2021-03-31 03:00:34,168] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: ingestion-pipeline.IngestingData 2021-03-31T06:57:05.709386+00:00 [queued]>
[2021-03-31 03:00:34,187] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: ingestion-pipeline.IngestingData 2021-03-31T06:57:05.709386+00:00 [queued]>
[2021-03-31 03:00:34,187] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-31 03:00:34,187] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-31 03:00:34,188] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-31 03:00:34,210] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): IngestingData> on 2021-03-31T06:57:05.709386+00:00
[2021-03-31 03:00:34,215] {standard_task_runner.py:52} INFO - Started process 15972 to run task
[2021-03-31 03:00:34,226] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'ingestion-pipeline', 'IngestingData', '2021-03-31T06:57:05.709386+00:00', '--job-id', '10', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ingestion.py', '--cfg-path', '/tmp/tmp2_xfwlv5', '--error-file', '/tmp/tmp0j5y37ys']
[2021-03-31 03:00:34,227] {standard_task_runner.py:77} INFO - Job 10: Subtask IngestingData
[2021-03-31 03:00:34,264] {logging_mixin.py:104} INFO - Running <TaskInstance: ingestion-pipeline.IngestingData 2021-03-31T06:57:05.709386+00:00 [running]> on host prathamesh-VirtualBox
[2021-03-31 03:00:34,321] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingestion-pipeline
AIRFLOW_CTX_TASK_ID=IngestingData
AIRFLOW_CTX_EXECUTION_DATE=2021-03-31T06:57:05.709386+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-31T06:57:05.709386+00:00
[2021-03-31 03:00:34,321] {python.py:118} INFO - Done. Returned value was: None
[2021-03-31 03:00:34,323] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=ingestion-pipeline, task_id=IngestingData, execution_date=20210331T065705, start_date=20210331T070034, end_date=20210331T070034
[2021-03-31 03:00:34,355] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-31 03:00:34,384] {local_task_job.py:146} INFO - Task exited with return code 0
